test/jdk/java/util/Arrays/SortingDeoptimizationTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @build SortingDeoptimizationTest
+ * @run main/othervm -Xcomp -ea -esa -XX:CompileThreshold=25 -XX:+UnlockExperimentalVMOptions -server -XX:-TieredCompilation -XX:+DeoptimizeALot SortingDeoptimizationTest 1e-2 100 50
```

Comment:
The test fails with release builds because `VM option 'DeoptimizeALot' is develop and is available only in debug version of VM.`.

Reply:
The original test (Sorting.java) does not have the `DeoptimizeALot` and other additional flags like `-Xcomp -ea -esa -XX:CompileThreshold=100` , etc. The other vmoptions were passed as additional flags during specific tests. Could we follow the same approach for this deoptimization test as well and limit the number of default flags?

Reply:
Please see the latest commit which fixed the release mode failure by adding `-XX:+IgnoreUnrecognizedVMOptions`.